driver/gl/window.go

```diff
@@ -60,6 +60,34 @@ func (w *window) SetFullScreen(full bool) {
 	})
 }
 
+func (w *window) CenterOnScreen() {
+	runOnMainAsync(func() {
+		// get window dimensions in pixels
+		monitor := getMonitorForWindow(w.viewport)
+		monMode := monitor.GetVideoMode()
+
+		// these come into play when dealing with multiple monitors
+		monX, monY := monitor.GetPos()
+		
+		// get current size of content inside the window
+		winContentSize := w.Content().MinSize()
```

Comment:
You can’t compare this directly with the window size as the content may be scaled (try using FYNE_SCALE=2.5). If you must compare the content size then multiply it by the canvas scale.